package.json

```diff
@@ -12,6 +13,7 @@
     "kill-port": "^2.0.1",
     "lint-staged": "^13.3.0",
     "prettier": "^2.8.8",
+    "rollup": "3.29.4",
```

Comment:
This is an indirect dependency and AFAIK we don't need to have it explicitly mentioned here.